src/Commands/MakeUser.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Portable\FilaCms\Commands;
+
+use Spatie\Permission\Traits\HasRoles;
+use Illuminate\Console\Command;
+use Filament\Models\Contracts\FilamentUser;
+use Spatie\Permission\Models\Role;
+use Schema;
+use Hash;
+
+class MakeUser extends Command
+{
+    protected $signature = 'fila-cms:make-user {--dry-run}';
+
+    protected $description = 'Creates a new user with an admin role';
+
+    public function handle()
+    {
+        $dryRun = $this->option('dry-run');
+
+        // check if Role table exists
+        if (Schema::hasTable('role') === FALSE) {
+
+        }
+
+        // check first if there's an admin role in place
+        $checkRole = Role::where('name', 'Admin')->first();
+
+        if (is_null($checkRole) === TRUE) { // no role, prompt the user to install
+            $this->error('Fila CMS not detected');
+            $this->info('Please install the Fila CMS installation command first');
+        }
+
+        
+        $userModelPath = 'App\Models\User';
+        $userModel = new $userModelPath;
+        $userFieldsRaw = Schema::getColumnListing($userModel->getTable());
+        
+        $excludeFields = [ 'id', 'created_at', 'updated_at', 'deleted_at', 'remember_token', 'email_verified_at' ];
+        $userFields = array_diff($userFieldsRaw, $excludeFields);
+
+        foreach ($userFields as $key => $field) {
+            $userModel->{$field} = $this->ask('Enter admin ' . $field);
+        }
+
+        // check for password field
+        if (in_array('password', $userFieldsRaw)) {
+            $userModel->password = Hash::make($userModel->password); // hash the password before saving
+        }
+
+        // auto-populate other fields
+        if (in_array('email_verified_at', $userFieldsRaw)) {
+            $userModel->email_verified_at = now();
+        }
+
+        // get the role
+        $role = Role::where('name', 'Admin')->first();
```

Comment:
We shouldn't need to retrieve this again, you've already got it from $checkRole on line 28